SalesforceSDK/Salesforce.SDK.SmartStore/Source/Store/SmartStore.cs

```diff
@@ -695,23 +695,20 @@ private object GetObject(SQLiteStatement statement, int position)
             {
                 return statement.GetText(position);
             }
-            catch (Exception e)
-            {
-            }
+            catch (Exception) { }
```

Comment:
We're using the Try-Catch approach here to determine if the object is Text/Int/Float. I don't think there is a need to log anything here every time the method is called. The log statement will not provide any real added value. But there may be a better approach than Try-Catch. @kchitalia will be looking at this file to clean up. I think we can get the data type using statement.DataType and using a switch statement we can determine which of GetText/GetInteger/GetFloat to call without needing any Try-Catch